docs/ci/codemagic.mdx

```diff
@@ -152,14 +152,87 @@ You can find a working example in this repository:
 
   </TabItem>
   <TabItem value="windows" label="Windows" default>
-The documentation for Windows is under development.
-<a href="https://github.com/shorebirdtech/docs/issues/98">
-Contributions welcome</a>.
+
+1. Add the group "shorebird" to your environment groups. If you're using a
+   different group name, use that instead. Also add a variable named `shorebird`
+   with the path to the Shorebird executable.
+
+```yaml
+workflows:
+  example:
+    name: Example
+    environment:
+      groups:
+        # Exports the SHOREBIRD_TOKEN environment variable
+        - shorebird
+      flutter: stable
+      vars:
+        # Workaround for
+        # https://github.com/orgs/codemagic-ci-cd/discussions/1921#discussioncomment-6582318
+        shorebird: "C:\\Users\\builder\\.shorebird\\bin\\shorebird.ps1"
+```
+
+2. Add a script to your workflow to set up Shorebird. This script will install
+   Shorebird.
+
+```yaml
+scripts:
+  - name: 🐦 Setup Shorebird
+    script: |
+      Set-ExecutionPolicy RemoteSigned -scope CurrentUser # Needed to execute remote scripts
+      iwr -UseBasicParsing 'https://raw.githubusercontent.com/shorebirdtech/install/main/install.ps1'|iex
+```
+
+3. Run Shorebird commands with `& $env:shorebird` instead of just `shorebird` as a workaround for [codemagic-ci-cd/discussions/1921](https://github.com/orgs/codemagic-ci-cd/discussions/1921#discussioncomment-6582318).
+
+```yaml
+scripts:
+  - name: 🚀 Shorebird Patch
+    script: |
+      & $env:shorebird patch android --force
```

Comment:
```suggestion
      & $env:shorebird patch android --release-version <RELEASE-VERSION> --force
```

The latest version requires users to specify which release version they are patching in order for shorebird to know which revision of Flutter to use.

Reply:
Ah, good to know. I created #110 to adjust the "Advanced example". For the windows tab I'm going to show the easy case, so I replaced `patch` with `release`.